src/molecules/formfields/shared/formfields.module.scss

```diff
@@ -407,7 +407,7 @@ $focused-border-color: color('brand-3');
     }
 
     span {
-      margin-top: ru(.125);
+      margin-top: 0;
```

Comment:
Does the `span` come with a margin by default? Can we just remove this styling altogether?